tck/src/main/java/org/eclipse/microprofile/lra/tck/service/LRAMetricType.java

```diff
@@ -19,26 +19,20 @@
  *******************************************************************************/
 package org.eclipse.microprofile.lra.tck.service;
 
-import org.eclipse.microprofile.lra.annotation.LRAStatus;
-import org.eclipse.microprofile.lra.annotation.ParticipantStatus;
-
 public enum LRAMetricType {
```

Comment:
I am pretty sure there is some code somewhere in the TCK that requires the enum values to match LRAStatus enum names. Can you reinstate the enum values please.

Reply:
@mmusgrov I can, but since all values were just copying the same names as the enum values why can't just use enum's `name()` method?

Reply:
@mmusgrov I put my comment to the wrong place. I would like mention that current implementation of `LRAMetricType` uses the constructor in form of
```
LRAMetricType(String label) {
}
```
which means the enum needs to define the `label` as a `String` but the label is lost after construction. There is no reason to have that string representations in the enum class when they are not used and they can't be used either.

Reply:
@mmusgrov did that answer your point? Can we merge this?